### ISSUE TITLE
Wget/Curl error shown for RUN commands without either

### ISSUE THREAD

**Issue by egeriis** [hadolint/hadolint#368]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior

Error should not be shown in blocks that are not affected by the error in question.

### Actual behavior

In this case an error is shown for RUN commands that doesn't use curl nor wget, an error which is triggered from two other RUN commands using one and the other.

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

This is reproducible in current online version of hadolint.

**Dockerfile** (if relevant)

```Dockerfile
FROM image:1.0

USER root

RUN set -ex \
  && wget http://pear.php.net/go-pear.phar

RUN set -ex \
  && curl -q -o composer.phar https://getcomposer.org/download/"${PHP_VERSION_COMPOSER}"/composer.phar \
  && chmod +x composer.phar \
  && ./composer.phar install --dev --no-interaction --no-autoloader \
  && ./composer.phar dumpautoload

## Copy build output so we can access it after mounting over it
RUN mkdir /build \
 && cp -r /app/vendor /build/vendor \
 && chown -R www-data:www-data /app/vendor
```

**Additional environment details (OS, stack version, etc.)**
